### DISCUSSION TITLE
Silent - One line installation

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#4238] [Unsupported Versions] [answered]
Hello,

I'm looking to install winlogbeat and ossec in one line with the SO as target.
I can't find any doc on internet to do it. Any idea?

I'm trying to auto install SecurityOnion 2 and agent on the client Linux/Windows but i can't find how i can install Ossec or Winlogbeat in 1 command line without any pop up or other configuration.

**Comment by b4b857f6ee** [comment]
Any idea? :/

**Comment by weslambert** [comment, marked as answer]
You should be able to register a Wazuh agent using the registration service, or the API, as described here (assuming the port is open from the Security Onion side by using so-allow):

https://documentation.wazuh.com/3.13/user-manual/registering/
